docs/field-details.asciidoc

```diff
@@ -3853,6 +3853,21 @@ The OS fields contain information about the operating system.
 
 // ===============================================================
 
+| os.commercial_family
+| Categorize the operating system in one of the broad commercial families.
+
+One of these following values should be used (lowercase): linux, macos, unix, windows.
```

Comment:
maybe we should also make "other" a legal value..for any edge cases where someone out there has data from something we could not anticipate

Reply:
Here I'd rather add specific guidance to "not populate the field if none of the options fit, but please let us know". Then we keep adding allowed values as needed.